src/components/MultiHopTrade/components/TradeInput/components/Claim/ClaimStatus.tsx

```diff
@@ -0,0 +1,154 @@
+import { CheckCircleIcon, WarningTwoIcon } from '@chakra-ui/icons'
+import { Button, CardBody, CardFooter, Center, Heading, Link, Stack } from '@chakra-ui/react'
+import { TxStatus } from '@shapeshiftoss/unchained-client'
+import { AnimatePresence } from 'framer-motion'
+import React, { useCallback, useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { useHistory } from 'react-router'
+import { Amount } from 'components/Amount/Amount'
+import { CircularProgress } from 'components/CircularProgress/CircularProgress'
+import { SlideTransition } from 'components/SlideTransition'
+import { SlideTransitionY } from 'components/SlideTransitionY'
+import { queryClient } from 'context/QueryClientProvider/queryClient'
+import { getTxLink } from 'lib/getTxLink'
+import { fromBaseUnit } from 'lib/math'
+import { selectAssetById } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import type { ClaimDetails } from './hooks/useArbitrumClaimsByStatus'
+import { ClaimRoutePaths } from './types'
+
+type ClaimStatusBodyProps = {
+  amountCryptoPrecision: string
+  description: string
+  icon: JSX.Element
+  status: TxStatus
+  symbol: string
+  title: string
+}
+
+export const ClaimStatusBody: React.FC<ClaimStatusBodyProps> = ({
+  amountCryptoPrecision,
+  description,
+  icon,
+  status,
+  symbol,
+  title,
+}) => {
+  return (
+    <SlideTransitionY key={status}>
+      <CardBody py={32}>
+        <Center flexDir='column' gap={4}>
+          {icon}
+          <Stack spacing={2} alignItems='center'>
+            <Heading as='h4'>{title}</Heading>
+            <Amount.Crypto
+              prefix={description}
+              value={amountCryptoPrecision}
+              symbol={symbol}
+              color='text.subtle'
+            />
+          </Stack>
+        </Center>
+      </CardBody>
+    </SlideTransitionY>
+  )
+}
+
+type ClaimStatusProps = {
+  activeClaim: ClaimDetails
+  claimTxHash: string
+  claimTxStatus: TxStatus
+}
+
+export const ClaimStatus: React.FC<ClaimStatusProps> = ({
+  activeClaim,
+  claimTxHash,
+  claimTxStatus,
+}) => {
+  const history = useHistory()
+  const translate = useTranslate()
+
+  const handleGoBack = useCallback(async () => {
+    await queryClient.invalidateQueries({
```

Comment:
why you no work?

Reply:
https://github.com/shapeshift/web/pull/7563/commits/344a991d0488d2e33ca54467763cce0ad05584a0